### DISCUSSION TITLE
Does it possible to disable validation in autosave ?

### DISCUSSION THREAD

**Discussion by slavb18** [vazco/uniforms#1075] [Q&A] [answered]
Hi!

Does it possible to disable validation in autosave ?
E.g. partial form could be autosaved, even not valid, while user enters data,but in the end when users submits data, it should be validated

**Comment by radekmie** [comment, marked as answer]
Hi @slavb18! There's no option to disable validation in autosave and we don't plan to have such a feature at all. However, your use case doesn't need that. What you can do instead, is to get the form model manually (`form.getModel()`) either on some user interaction (e.g., "Save" button click) or per-change, with `onChange`/`onChangeModel`. This way you can persist the (partial) model without validation.

Let me know whether it's clear.

**Comment by slavb18** [comment]
Is there plans to make submit button with name/value pair ?
e.g 
submitField={() => &lt;button name="key" value="value">submit!&lt;/button>}

in html form this leads to different data submitted from different submit buttons

this could be use also to distinguish autosave from submit button click (in second case model will contain key:"value" pair)

**Reply by radekmie** [reply]
@slavb18 I wasn't even aware of this API! Well, no, it's not supported out of the box nor I don't think we'll implement that. However, you can create a subclass of the form component and extract it from the `event` directly.
